content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -360,8 +360,14 @@ with `phase: "Succeeded"`.
 There are situations where you want to fail a Job after some amount of retries
 due to a logical error in configuration etc.
 To do so, set `.spec.backoffLimit` to specify the number of retries before
-considering a Job as failed. The back-off limit is set by default to 6. Failed
-Pods associated with the Job are recreated by the Job controller with an
+considering a Job as failed.
+
+The `.spec.backoffLimit` is set by default to 6, unless the
+[backoff limit per index](#backoff-limit-per-index) is specified - when
+`.spec.backoffLimitPerIndex` is specified, then it is defaults to 2147483647
+(MaxInt32).
```

Comment:
```suggestion
[backoff limit per index](#backoff-limit-per-index) is specified. When
`.spec.backoffLimitPerIndex` is specified, then `.spec.backoffLimit` defaults to 2147483647
(MaxInt32).
```

might be easier to read?

Reply:
I'd probably go even a bit further similarly to my suggestion from https://github.com/kubernetes/kubernetes/pull/131256#discussion_r2039952691 and make it more explicit:
```
backoff limit per index](#backoff-limit-per-index) is specified (only Indexed Job).
When `.spec.backoffLimitPerIndex` is specified, then `.spec.backoffLimit` defaults 
to 2147483647 (MaxInt32).
```

Reply:
Addressed, used the second suggestion, but both are good. Thanks!